open-company/open-company-web
# Sections picker restyle

Card: https://trello.com/c/CxWpmIgI

Abstract: https://share.goabstract.com/58e7f0a7-9cf3-4201-bc8a-e6ed933e29a0

To test:
- try to add a new section on desktop
- while editing a post pick an existing section
- while editing a post add a new section
- publish the post and make sure the section is visible in the left nav
- repeat the last 3 steps on mobile
- on desktop make sure that you can add a private section and add users with a note
- make sure you can enable slack mirror and it gets saved
